mail/protocols/imap/src/main/java/com/fsck/k9/mail/store/imap/SearchResponse.kt

```diff
@@ -0,0 +1,43 @@
+package com.fsck.k9.mail.store.imap
+
+internal class SearchResponse private constructor(
```

Comment:
This class can be migrated to a data class:
```suggestion
@ConsistentCopyVisibility
internal data class SearchResponse private constructor(
```